static/src/js/components/PreviewMapLink/PreviewMapLink.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+class PreviewMapLink extends React.Component {
```

Comment:
Is this deep's changes?  If so you should rebase, I think these changes will disapper